schema/show.js

```diff
@@ -91,6 +95,25 @@ const ShowType = new GraphQLObjectType({
         return fetch.then(exclude(options.exclude, "id"))
       },
     },
+    artworks_connection: {
+      type: artworkConnection,
+      args: pageable(artworksArgs),
+      resolve: (show, options) => {
+        const path = `partner/${show.partner.id}/show/${show.id}/artworks`
+        const gravityOptions = parseRelayOptions(options)
+        delete gravityOptions.page
+
+        return total(path, Object.assign({}, gravityOptions, { size: 0 })).then(count => {
```

Comment:
Kind of gross thing here, resorting to getting the total count separately from the actual fetch. This doesn't feel like it totally warrants rewriting the `http` loader to work with headers, but I suppose that could be argued. Another option would be writing a `httpWithHeaders` loader or something similar (since we would want to cache these requests if they are unauthenticated).